### ISSUE TITLE
Update README with YAML formatted examples

### ISSUE THREAD

**Issue by alykhank** [msanders/cider#26]
Add example `bootstrap.yaml` and `defaults.yaml` files to README to demonstrate new syntax.

**Comment by RogerThiede** [comment]
I'm interested in this as well.  I'm curious if my [bootstrap.yaml](https://github.com/RogerThiede/dotfiles/blob/0af732309962cac6b83b2a0c188ab5e436d54d33/bootstrap.yaml) is normal or not since it's cluttered with ```!!python/unicode```.

**Comment by alykhank** [comment]
@RogerThiede I noticed the same thing, and I believe that it may be due to the `allow_unicode=True` flag to `yaml.dump()` in [cider/_sh.py#L213-L227](https://github.com/msanders/cider/blob/6ad279c60dab0382406ba5ddb784b4fb1c779bf7/cider/_sh.py#L213-L227). According to the [PyYAML documentation](http://pyyaml.org/wiki/PyYAMLDocumentation#StringconversionPython2only) this seems to be expected behaviour.

**Comment by RogerThiede** [comment]
Was [Prevent odd formatting of YAML files](https://github.com/msanders/cider/commit/afd2d091b2070db70e87b939157b2bfb3ce921bd) clobbered?  It's not reachable on master branch.

**Comment by msanders** [comment]
Yes, turns out that didn't prevent the spammy output. Not sure what's causing this.

**Comment by robsonpeixoto** [comment]
About the `!!python/unicode`. To fix it, I forced the create of the .json file with the command `touch ~/.cider/bootstrap.json`

**Comment by RogerThiede** [comment]
Thanks @robsonpeixoto!  I just tried that and Cider is now using the json file.
Instructions like that in the README file would satisfy this filed bug.
